drivers/spi/spi.c

```diff
@@ -85,3 +85,34 @@ int32_t spi_write_and_read(struct spi_desc *desc,
 {
 	return desc->platform_ops->write_and_read(desc, data, bytes_number);
 }
+
+/**
+ * @brief  Iterate over head list and send all spi messages
+ * @param desc - The SPI descriptor.
+ * @param head - First link of the list with messages to be sent
+ * @return SUCCESS in case of success, negativ error code otherwise.
+ */
+int32_t spi_transfer(struct spi_desc *desc,
```

Comment:
I think the name and/or the description should better emphasize that this is a continuous transfer of multiple messages.

Reply:
I understand due to the other discussion that this is not necessarily the case. Resolved.